recipes/wineditline/all/CMakeLists.txt

```
@@ -1,8 +1,17 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
+project(WinEditLine C)
 
-project(WinEditLineWrapper C)
+add_library(edit
+    src/editline.c
+    src/fn_complete.c
+    src/history.c
+    src/libedit.def
+)
+target_include_directories(edit PRIVATE src)
```

```suggestion
target_include_directories(edit PRIVATE src)
set_target_properties(edit PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Export symbols on Windows to produce .DLL, only when building with BUILD_SHARED_LIBS:

https://cmake.org/cmake/help/latest/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html

---

Is it really needed? The shared library built and linked fine in CI.

---

It does not generate dll at all: https://c3i.jfrog.io/c3i/misc/logs/pr/18652/5-windows-visual_studio/wineditline/2.206//d057732059ea44a47760900cb5e4855d2bea8714-build.txt